.github/workflows/ci-storage.yml

```diff
@@ -53,54 +57,23 @@ jobs:
       BINTRAY_PASS: ${{ secrets.BINTRAY_PASS }}
     steps:
       - name: Checkout
-      - uses: actions/checkout@v2
-      - uses: coursier/cache-action@v4
-        with:
-          path: ~/.cache/coursier
-          key: ${{ runner.os }}-coursier-scala-${{ hashFiles('**/*.sbt') }}
-          restore-keys: ${{ runner.os }}-coursier-scala-
-      - uses: olafurpg/setup-scala@v5
+        uses: actions/checkout@v2
+      - name: Coursier Cache
+        uses: coursier/cache-action@v5
+      - name: Setup Scala
+        uses: olafurpg/setup-scala@v10
         with:
-          java-version: adopt@1.8-0
-      - name: PublishToBintray
+          java-version: adopt@1.11
```

Comment:
why java 11 for storage service?

Reply:
because it depends on rdf and kernel which require 11